sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -37,13 +39,42 @@ fun HomeScreen(
     val showLoading: IsLoading by viewModel.showLoading.collectAsState()
     val uiModels: List<UiModel> by viewModel.uiModels.collectAsState()
 
+    CameraPermission()
+
     HomeScreenContent(
         uiModels = uiModels,
         showLoading = showLoading,
         onItemClick = viewModel::navigateToSecond
     )
 }
 
+/**
+ * The permission check composable needs to be separate from the HomeScreenContent composable to avoid re-composition
```

Comment:
@luongvo Optional:

```suggestion
 * [CameraPermission] needs to be separate from [HomeScreenContent] to avoid re-composition
```

Would you like to update it? If not, I'll merge it now 🤖

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/375/commits/eb95572869dd994af2a445fe91e9c785d9ac9374